plat/xilinx/zynqmp/aarch64/zynqmp_common.c

```diff
@@ -59,6 +61,7 @@ static const struct {
 	unsigned int id;
 	unsigned int ver;
 	char *name;
+	bool evexists;
```

Comment:
nit: It's a bit weird to use a `bool` if you are going to assign to it integer values (1 or 0).

Reply:
Thanks for your comment, will use true or false instead of 1 or 0.